pex/environment.py

```diff
@@ -132,7 +132,9 @@ def can_add(self, dist):
   def activate(self):
     if not self._activated:
       with TRACER.timed('Activating PEX virtual environment from %s' % self._pex):
+        before = sys.path[:]
         self._working_set = self._activate()
+        sys.path = sys.path[len(before):] + before
```

Comment:
this feels like an odd place to arbitrarily mutate `sys.path`. ideally all sys.path mutation would take place in/around https://github.com/pantsbuild/pex/blob/master/pex/pex.py#L271